engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/desugar/LambdaShorthandToLambda.scala

```diff
@@ -123,7 +121,6 @@ case object LambdaShorthandToLambda extends IRPass {
     freshNameSupply: FreshNameSupply
   ): Expression = {
     ir.transformExpressions {
-      case asc: Type.Ascription => asc
```

Comment:
This is relatively nice pass, @Akirathan that could be good subject to
- #10981

This `case asc: Type.Ascription => asc` is currently a way to _"stop processing"_ the subtree.